test/jasmine/tests/plots_test.js

```diff
@@ -326,7 +383,7 @@ describe('Test Plots', function() {
         beforeEach(function(done) {
             gd = createGraphDiv();
 
-            Plotly.plot(gd, [{ x: [1, 2, 3], y: [2, 3, 4] }], {})
+            Plotly.plot(gd, [{ x: [1, 2, 3], y: [2, 3, 4] }], { autosize: true })
```

Comment:
@etpinard An alternative that would be fully backwards-compatible is that `Plot.resize(gd)` deletes `gd.layout.width` and `gd.layout.height` and calls `Plotly.relayout(gd, { autosize: true });`. I'll prepare a PR so that you can check that it'd fix the issue in the `mapbox` branch.